pkgs/tools/typesetting/tex/texlive/default.nix

```
@@ -111,6 +111,7 @@ let
         [( if (attrs.hasRunfiles or false) then mkPkgV "run"
             # the fake derivations are used for filtering of hyphenation patterns
           else { inherit pname version; tlType = "run"; }
```

```suggestion
          else { inherit pname version; tlType = "run"; hasHyphens = attrs.hasHyphens or false; }
```
Or we need it for mkPkgV branch as well? If yes, then I wonder why the comment implies that it's the tlType="run" packages that are used for hyphenations.

---

Good catch, this is a bug (even before the PR). The hyphenation patterns are in `tlType == "run"` only. I suggest we test for `tlType == "run"` in `combine.nix` (less performant maybe, but more robust).